tests/integration/targets/mso_schema/tasks/main.yml

```diff
@@ -18,8 +19,14 @@
       validate_certs: '{{ mso_validate_certs | default(false) }}'
       use_ssl: '{{ mso_use_ssl | default(true) }}'
       use_proxy: '{{ mso_use_proxy | default(true) }}'
-      output_level: '{{ mso_output_level | default("info") }}'
+      output_level: '{{ mso_output_level | default("debug") }}'
```

Comment:
Could we put this back to info if not needeed.

Reply:
ok